app/components/recognition/form.js

```diff
@@ -316,26 +332,11 @@ export default class FormComponent extends Component {
   }
 
   @action
-  async getAwardedBy() {
-    let awardedByValue = this.items[0];
-    if (this.currentRecognition.selectedItem !== this.items[0]) {
-      awardedByValue = this.currentRecognition.recognitionModel.awardedBy;
-    }
-
-    const administrativeUnits = await this.store.query('administrative-unit', {
-      filter: {
-        ':exact:name': awardedByValue,
-      },
-    });
-    let awardedBy = administrativeUnits?.[0];
-    if (!awardedBy) {
-      awardedBy = this.store.createRecord('administrative-unit', {
```

Comment:
I'm not sure why this was implemented this way but it seems unlikely we ever wanted this behavior? 

This code was linked to the input field that would be displayed if the user selected "Andere". It searches for an administrative unit with the exact name and if it doesn't exist, create a new one with that name.

Reply:
I think we can safely removed that code. AFAIK, only gemeentes or other official instances can give recognise.